packages/api-metadata/src/storage/fromMetadata/getHasher.ts

```diff
@@ -31,7 +32,20 @@ export default function getHasher (hasher?: StorageHasher): HasherFunction {
     return (data: HasherInput): Uint8Array => xxhashAsU8a(data, 256);
   }
 
-  // FIXME Add Twox128Concat
+  if (hasher.isTwox64Concat) {
+    return (data: HasherInput): Uint8Array => {
+      let dataAsU8a: Uint8Array;
```

Comment:
You could just use `const dataAsU8a = u8aTou8a(data)` and skip the ifs. (It does the u8a/string/buffer dance on your behalf) Apart form that, the logic matches.

Reply:
ah, I knew there was something like that